src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.php

```diff
@@ -119,4 +119,11 @@
             ])
             ->tag('validator.auto_mapper')
     ;
+
+    if (true === param('validator.disable_translation')) {
```

Comment:
the `validator.disable_translation` parameter is never set

Reply:
Oups that's right, I updated `FrameworkExtension.php` instead